### ISSUE TITLE
Pre-Schematron Structured FedRAMP Reviewer Validations Rule Data

### ISSUE THREAD

**Issue by ohsh6o** [18F/fedramp-automation#83]
As the FedRAMP PMO and ASAP P3 developer, in order to better understand how the FedRAMP Validations project correctly implements each business rule reviewers manually validate in my ongoing processes, I want to see a more structured list of these rules and how they map to the validation developers' code.

**Context:**

Onboarded 10x developers and FedRAMP stakeholders have discussed a need for more structured information than what was provided previously to the 10x ASAP P2 Team last year in the form of the [Agency Review Report Template](https://docs.google.com/spreadsheets/d/1Rcw7KvUudIPMSMmo0lbhXE3QgqqaS76S0BmFC8ThrXA/edit#gid=177951143). The current developers would like a more intentional, structured precursor document with a listing of rules that can be reviewed and modified over time. Once this precursor document is established, the developers would like to investigate more directly mapping it to the code outputs (in later development efforts, perhaps after Sprint 2), perhaps using it to programmatically generate tests and/or the validation code itself. 

Nice to have: A rendition of this structured precursor document into HTML or an alternative presentation format with the FART and JAB team would be ideal.

**Acceptance:**

- [ ] Documentation of approach and a minimal example of one structured business rule with required data
- [ ] A structured presentation of our understanding on FedRAMP business rules
- [ ] Document our understanding of what triggers changes in FedRAMP business rules (i.e. what drives FedRAMP rule changes): 
  - NIST baseline changes (OMB mandates this adoption)
  - OSCAL releases? Is this just a cascade effect to NIST change, specimen SSP
  - Internal process improvement

**Dependencies:**
- [ ] A working agreement with FedRAMP FART and JAB contacts to align on this structured data

**Comment by ohsh6o** [comment]
Will close this in favor of 18F/fedramp-automation#65.

**Comment by ohsh6o** [comment]
Closed the wrong one, will fix later.

**Comment by ohsh6o** [comment]
Now that I fixed this one, will close 65. I had intended to keep this one after all.

**Comment by GaryGapinski** [comment]
Need associated documentation and working agreements

**Comment by GaryGapinski** [comment]
I'll create a document to explain the constructs.

The document names are IMO a bit too simple and could be improved. Suggestions are welcome.

**Comment by GaryGapinski** [comment]
See also #198.

**Comment by sstatz** [comment]
@ohsh6o  to coordinate a conversation to unblock.  See Preconditions.

**Comment by ohsh6o** [comment]
Per 18F#122, we will include this in today's user feedback session. Will loop in feedback on how to progress with an understanding of this rules definition system after our first trial run.

**Comment by ohsh6o** [comment]
Will talk to FART leads as our next review session will be out of this sprint and determine how we should move forward this approach depending on feedback.

**Comment by ohsh6o** [comment]
Discussion with FART lead today, will further discuss this work during briefing with FedRAMP PMO on Thursday (after sprint, was rescheduled from Monday).
